.github/workflows/release-packages.yaml

```diff
@@ -44,6 +44,47 @@ jobs:
           asset_name: ${{ steps.create_packages.outputs.deb_package_name }}
           asset_content_type: application/x-deb
 
+  ubuntu-18_04-package:
+    runs-on: ubuntu-18.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Fetch dependencies
+        run: sudo apt install g++ flex bison cmake ninja-build maven jq libxml2-utils dpkg-dev
+      - name: Configure CMake
+        run: |
+          mkdir build
+          cd build
+          cmake .. -G Ninja -DCMAKE_BUILD_TYPE=Release
+      - name: Build using Ninja
+        run: |
+          cd build
+          ninja
+      - name: Run CTest
+        run: cd build; ctest . -V -L CORE -C Release
+      - name: Create packages
+        id: create_packages
+        run: |
+          cd build
+          ninja package
+          deb_package_name="$(ls *.deb)"
+          echo "::set-output name=deb_package::./build/$deb_package_name"
+          echo "::set-output name=deb_package_name::ubuntu-18.04-$deb_package_name"
+      - name: Get release info
+        id: get_release_info
+        uses: bruceadams/get-release@v1.2.0
+      - name: Upload binary packages
+        uses: actions/upload-release-asset@v1
+        with:
+          upload_url: ${{ steps.get_release_info.outputs.upload_url }}
+          asset_path: ${{ steps.create_packages.outputs.deb_package }}
+          asset_name: ${{ steps.create_packages.outputs.deb_package_name }}
+          asset_content_type: application/x-deb
+
+
```

Comment:
Seems a bit of a shame that so much of this is a duplication of the the ubuntu-20_04 target - is there any way to re-use this (I don't know github actions, sorry! - but was thinking about either some way of 'including' comment parts, or moving those out into a script/cmake action?)

Reply:
@chrisr-diffblue There is, technically. Although to be honest I've been deliberately avoiding it, I find these short _enough_ and I find logic gets harder to follow as you split it over multiple scripts. This isn't enough script for this to really save us any work, so the main advantage would be it's easier to see that these are doing mostly the same thing – at the cost of hiding what that thing is behind possibly multiple files.